src/main/java/com/adobe/cq/testing/client/CQClient.java

```diff
@@ -205,6 +205,26 @@ public Boolean call() throws ClientException {
         return deletePagePolling.response;
     }
 
+    /**
+     * Returns whether a CQ page exists or not
+     */
+    public boolean exists(String pagePath) throws ClientException {
```

Comment:
Well, I originally intended exactly that - an override - but it's wrong, because it changes the scope of the method (i. e. it would only work for pages, not any path)